servant/src/Servant/API/Empty.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+module Servant.API.Empty(EmptyAPI(..)) where
+
+import           Data.Typeable    (Typeable)
+import           Prelude ()
+import           Prelude.Compat
+
+-- | An empty API: one which serves nothing.
```

Comment:
This has always been "a problem" with servant: terms in our DSL are just symbolic, so whenever we talk about their semantics, we're really talking about the semantics of their various interpretations.